docker/sbtc/signer/signer-config.toml

```diff
@@ -26,7 +26,7 @@
 # Environment: SIGNER_EMILY__ENDPOINTS
 # Environment Example: '{"endpoint":"https://api.emilyexample.com","api_key":"1234567890abcdef"},..'
 endpoints = [
-    { endpoint = "http://emily-server:3031" },
+    { endpoint = "http://localhost:3031" },
```

Comment:
I'll revert this before an eventual merge, comment so I don't forget.

Reply:
reverted in https://github.com/stacks-network/sbtc/pull/811/commits/f14503f8fc405b5995400433c7b477df82198cf3